### PR TITLE
Feat/bold titles

### PR DESCRIPTION
**Pull request by amysiu1028**
✨ What I did:
Bold texts, updated dom, css, and html

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [x]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [ ]  The code is DRY. If it's not, I tried my best
- [ ]  I reviewed my code before pushing

✨ What's next?
POST requests

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -31,30 +31,32 @@ export const displayUserInfo = (currentUser, activityToday) => {
...
 export const displayWaterInfo = (todaysHydrationDate, waterChartToDom) => {
-  waterConsumedToday.innerText = `Water Consumed Today: ${todaysHydrationDate.numOunces}oz`
+  waterConsumedToday.innerHTML = `<strong> Water Consumed Today:  </strong> ${todaysHydrationDate.numOunces} ounces`
   waterChart.innerHTML = waterChartToDom;
 };
  
 //Sleep info:
 export const displaySleepInfo = (averageSleep, aveSleepQuality, sleepToday, sleepChartToDom) => {
-  averageSleepOverall.innerText = `Average Overall Sleep: ${averageSleep} hours per day and Average Sleep Quality: ${aveSleepQuality}`;
+  averageSleepOverall.innerHTML = `<strong> Average Overall Sleep: </strong> ${averageSleep} hours per day and Average<br> <strong> Sleep Quality: </strong> ${aveSleepQuality}`;
```
Comment (ben-rosner-williamsburg) [inline src/domUpdates.js:40]: We need opening and closing tags for the rest of the text that isn't bolded.

File: src/domUpdates.js [hunk 2]
```diff
@@ -31,30 +31,32 @@ export const displayUserInfo = (currentUser, activityToday) => {
 
 // Water info:
 export const displayWaterInfo = (todaysHydrationDate, waterChartToDom) => {
-  waterConsumedToday.innerText = `Water Consumed Today: ${todaysHydrationDate.numOunces}oz`
+  waterConsumedToday.innerHTML = `<strong> Water Consumed Today:  </strong> ${todaysHydrationDate.numOunces} ounces`
```
Comment (ben-rosner-williamsburg) [inline src/domUpdates.js:34]: We need opening and closing tags for the rest of the text that isn't bolded.

File: src/domUpdates.js [hunk 3]
```diff
@@ -31,30 +31,32 @@ export const displayUserInfo = (currentUser, activityToday) => {
...
   sleepChart.innerHTML = sleepChartToDom;
 };
   
 //Activity info:
 export const displayActivityInfo = (milesPerDay, minutesPerDay, activityToday, activityChartToDom) => {
-    stepsMadePerDay.innerText = `Total user's steps today: ${activityToday.numSteps} steps.`;
-    minutesActivePerDay.innerText = `Total active minutes today: ${minutesPerDay} minutes today.`;
-    milesWalkedPerDay.innerText = `User has walked ${milesPerDay} miles.`
+    stepsMadePerDay.innerHTML = `<strong> Total user's steps today: </strong> ${activityToday.numSteps} steps.`;
+    minutesActivePerDay.innerHTML = `<strong> Total active minutes today: </strong> ${minutesPerDay} minutes today.`;
```
Comment (ben-rosner-williamsburg) [inline src/domUpdates.js:48]: We need opening and closing tags for the rest of the text that isn't bolded.

**Review by ben-rosner-williamsburg — COMMENTED** [review]
Hi! Can we add opening and closing tags for the innerHTML so that all text is formatted properly. Thanks!

**Review by ben-rosner-williamsburg — APPROVED** [review]
Awesome. Looks good. I'll approve this.

File: src/domUpdates.js (hunk 3 above)
Comment (amysiu1028) [reply src/domUpdates.js:48]: I just added it closing and opening tags, and added tabbing on this branch

**Pull request merged**